hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionNameWithMetrics.java

```diff
@@ -18,7 +18,7 @@
 
     @JsonCreator
     public SubscriptionNameWithMetrics(
-            @JsonProperty("topicQualifiedName") String topicQualifiedName,
+            @JsonProperty("topicName") String topicQualifiedName,
```

Comment:
Changed so that the attribute name is equal to that present in Subscription class. This allows filtering by topicName for both classes.

Reply:
Should we change variable name too? Or create a separate variable with the name `topicName` ?

Reply:
I am not sure but:
1) I did not want to change the api 
2) Variable _topicName_ would be exactly the same as _topicQualifiedName_
A similar thing was done in _Topic_ as well as _Subscription_ .

Reply:
Isn't changing `JsonProperty` already an API change? ;)

Reply:
Yes, it is - I used wrong wording. This change is somehow less invasive.

Reply:
I agree that this change is not so invasive, let’s try it +1